### PR TITLE
Add IDAM-WAPR view

### PR DESCRIPTION
**Pull request by henrydobson**
Add IDAM-WAPR view to cac-prod

### CODE REVIEW

File: cac-prod.yml [hunk 1]
```diff
@@ -169,6 +169,12 @@ jenkins:
         name: RD
         recurse: true
         title: Ref Data Dashboard
+    - buildMonitor:
+        includeRegex: >-
+          HMCTS_.*IDAM.*\/(idam-api|idam-web-public|idam-web-admin)\/PR.*
```
Comment (doncem) [inline cac-prod.yml]: perhaps `idam-*` is enough

Comment (henrydobson) [reply cac-prod.yml]: These three builds currently cannot run in parallel. Whilst that is being addressed, this visibility helps cause less concurrency issues. `idam-*` matches too many projects.

Comment (henrydobson) [reply cac-prod.yml]: @doncem please resolve or continue this discussion.

Comment (doncem) [reply cac-prod.yml]: I'm not the owner of repo nor IDAM project - not aware what's needed. I merely reviewed as outsider here giving extra thought 😉

**Pull request closed without merging**